asapdiscovery-docking/asapdiscovery/docking/cli.py

```diff
@@ -47,92 +43,43 @@ def cli():
     default=500,
     help="The maximum number of docking results to return, ordered by docking score",
 )
-@click.option(
-    "--use-dask",
-    is_flag=True,
-    default=False,
-    help="Whether to use dask for parallelism.",
-)
-@click.option(
-    "--dask-type",
-    type=click.Choice(DaskType.get_values(), case_sensitive=False),
-    default=DaskType.LOCAL,
-    help="The type of dask cluster to use. Can be 'local', 'lilac-cpu' or  'lilac-gpu'.",
-)
 @click.option(
     "--posit-confidence-cutoff",
     type=float,
     default=0.7,
     help="The confidence cutoff for POSIT results to be considered",
 )
-@click.option(
-    "--output-dir",
-    type=click.Path(resolve_path=True, exists=False, file_okay=False, dir_okay=True),
-    help="The directory to output results to.",
-    default="output",
-)
-@click.option(
-    "--input-json",
-    type=click.Path(resolve_path=True, exists=True, file_okay=True, dir_okay=False),
-    help="Path to a json file containing the inputs to the docking workflow,  WARNING: overrides all other inputs.",
-)
-@click.option(
-    "-l",
-    "--ligands",
-    type=click.Path(resolve_path=True, exists=True, file_okay=True, dir_okay=False),
-    help="Path to a file containing a list of complexes to dock.",
-)
-@click.option(
-    "--fragalysis-dir",
-    type=click.Path(resolve_path=True, exists=True, file_okay=False, dir_okay=True),
-    help="Path to a directory containing fragments to dock.",
-)
-@click.option(
-    "--structure-dir",
-    type=click.Path(resolve_path=True, exists=True, file_okay=False, dir_okay=True),
-    help="Path to a directory containing structures to dock instead of a full fragalysis database.",
-)
-@click.option(
-    "--postera-molset-name",
-    type=str,
-    default=None,
-    help="The name of the molecule set to pull from and upload to.",
-)
-@click.option(
-    "--du-cache",
-    type=click.Path(resolve_path=True, exists=True, file_okay=False, dir_okay=True),
-    help="Path to a directory where design units are cached.",
-)
-@click.option(
-    "--gen-du-cache",
-    type=click.Path(resolve_path=True, exists=False, file_okay=False, dir_okay=True),
-    help="Path to a directory where a design unit cache should be generated.",
-)
-@click.option(
-    "--ml-scorer",
-    type=click.Choice(
-        ASAPMLModelRegistry.get_implemented_model_types(), case_sensitive=True
-    ),
-    multiple=True,
-    help="The names of the ml scorer to use, can be specified multiple times to use multiple ml scorers.",
-)
+@ligands
+@postera_args
+@pdb_file
+@fragalysis_dir
+@structure_dir
+@gen_cache
+@cache_dir
+@cache_type
+@dask_args
+@output_dir
+@input_json
+@ml_scorer
 def large_scale(
-    postera: bool,
-    postera_upload: bool,
     target: TargetTags,
-    n_select: int,
-    top_n: int,
-    use_dask: bool,
-    dask_type: str,
+    n_select: int = 10,
```

Comment:
```suggestion
    n_select: int = 5,
```
5 is plenty for production and saves a whole lot of compute